app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Entities\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+
+class LoginController extends Controller
+{
+    public function index()
+    {
+        return view('login');
+    }
+    public function attemptLogin(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'username' => 'required',
+            'password' => 'required|min:3',
+        ]);
+
+        if ($validator->fails()) {
+            dd($validator->errors());
+            return back()->with('errors', $validator->errors());
+        }
+        if(!User::where('username', $request->get('username'))->first()){
+            return back()->with('error', 'No user found');
```

Comment:
Może zacznijmy uzywać translacji laravelowej?

Reply:
zrobiłem na to zadanie